### DISCUSSION TITLE
How to enable overlays for timelapse (working for video, not timelapse)

### DISCUSSION THREAD

**Discussion by akomakom** [gopro/labs#796] [Q&A]
Hi all,

I have a brand new Hero 11 and just installed the [labs firmware](https://github.com/gopro/labs/discussions?discussions_q=mburn+timelapse) today.   All I want is a 60s interval timelapse with the timestamp.

After some research I was able to get timestamp overlays in video recordings (If I set the scale above 30) using [this generator](https://gopro.github.io/labs/control/overlays/)
`https://gopro.github.io/labs/control/set/?cmd=g0oMBRNO=10oMBURN="(0,36)[TCHH:MM:SSaa mm-dd-yyyy ]"&title=Overlays`

However, no combination has yielded any text in timelapse recordings (I've tried 1080, 4k, 5k).  

I've also tried to combine the overlay code with the custom code using [this generator](https://gopro.github.io/labs/control/custom/):
`https://gopro.github.io/labs/control/set/?cmd=mTp1r4TfL0g0oMBRNO=10oMBURN="(0,36)[TCHH:MM:SSaa mm-dd-yyyy ]"`

(Still no effect).

I've also tried the QR code suggested [here](https://github.com/gopro/labs/issues/664#issuecomment-1841260492), and this one does not yield overlays in any mode.

Can someone provide a working QR link or point out what I'm doing wrong?  

Thanks!

**Comment by dnewman-gpsw** [comment]
Correct, it only work in limited Video modes, not Timelapse modes on HERO10, 11 & 12.  This is an old GP1 feature that isn't well supported on GP2 cameras. 

From the documentation (https://gopro.github.io/labs/control/overlays/):

```
Known Issues:
  HERO10/11/12 support is limited to video modes 4K 16:9 up to 30, 2.7K up to 60 and 1080p up to 120fps, in 8-bit only.
```

**Comment by akomakom** [comment]
I appreciate the answer.  The wording felt vague and I was hopeful.  I wound up using an old smartphone to do the timelapse.
